labelbox/schema/create_batches_task.py

```diff
@@ -0,0 +1,58 @@
+import json
+from typing import List, Optional, Dict, Any
+
+from labelbox.orm.model import Entity
+
+
+class CreateBatchesTask:
+
+    def __init__(self, client, project_id: str, batch_ids: List[str],
+                 task_ids: List[str]):
+        self.client = client
+        self.project_id = project_id
+        self.batches = batch_ids
+        self.tasks = [
+            Entity.Task.get_task(self.client, task_id) for task_id in task_ids
+        ]
+
+    def wait_till_done(self, timeout_seconds: int = 300) -> None:
+        """
+        Waits for the task to complete.
+
+        Args:
+            timeout_seconds: the number of seconds to wait before timing out
+
+        Returns: None
+        """
+
+        for task in self.tasks:
```

Comment:
[optional] how many tasks we expect here? would running these in parallel be worth doing it?

Reply:
10 max at the moment, we can run these in parallel as a follow-up if the limit is increased